File: uri/python/1038.py
# -*- coding: utf-8 -*-

codigo, quantidade = input().split(" ")

condigo = int(codigo)
quantidade = float(quantidade)
        
if codigo == '1':
    print('Total: R$ {:.2f}'.format(4.00 * quantidade))

elif codigo == '2':
    print('Total: R$ {:.2f}'.format(4.50 * quantidade))

elif codigo == '3':
    print('Total: R$ {:.2f}'.format(5.00 * quantidade))

elif codigo == '4':
    print('Total: R$ {:.2f}'.format(2.00 * quantidade))

elif codigo == '5':
    print('Total: R$ {:.2f}'.format(1.50 * quantidade))

File: uri/C/beginner/1038.c
#include <stdio.h>

int main()
{

	int codigo, quantidade;
	double pago;

	scanf("%d %d", &codigo, &quantidade);

	if(codigo == 1){
		pago = 4 * quantidade;
		printf("Total: R$ %.2lf\n", pago);
	}


	if(codigo == 2){
		pago = 4.50 * quantidade;
		printf("Total: R$ %.2lf\n", pago);	
	}


	if(codigo == 3){
		pago = 5 * quantidade;
		printf("Total: R$ %.2lf\n", pago);
	}


	if(codigo == 4){
		pago = 2 * quantidade;
		printf("Total: R$ %.2lf\n", pago);
	}


	if(codigo == 5){
		pago = 1.50 * quantidade;
		printf("Total: R$ %.2lf\n", pago);
	}

	return 0;
}
